### PR TITLE
reworked the second vignette

### PR DESCRIPTION
**Pull request by KKPMW**

### CODE REVIEW

**Comment by KKPMW** [comment]
Hello @christophsax and @a1exgo - updated the second vignette. Feel free to review and leave comments/suggestions.

**Comment by KKPMW** [comment]
I will still have to make a few adjustments after seeing if 'inst' scripts are still necessary

**Comment by KKPMW** [comment]
@a1exgo pinging to check if you got the previous notification

File: vignettes/daily-series.Rmd [hunk 1]
```diff
@@ -14,143 +14,140 @@ knitr::opts_chunk$set(
...
 proc_combine_freq("Rezession", "CH")
 ```
 
-##### Seasonal Adjustment
+### Step 3: seasonal adjustment ###
 
-- [ ] FXIME some text from the website
+Some keywords might display seasonal patterns.
+For example it is not surprising that searches for gardening are higher in spring.
+In order to make meaningful comparisons over time such seasonal patterns, present within the data, need to be removed.
```
Comment (a1exgo) [inline vignettes/daily-series.Rmd:112]:
```suggestion
In order to make meaningful comparisons over time such seasonal patterns, present within the raw data, need to be filtered out.
```

File: vignettes/daily-series.Rmd [hunk 2]
```diff
@@ -14,143 +14,140 @@ knitr::opts_chunk$set(
...
+To seasonally adjust a combined keyword, the script calls the following internal function:
 
 ```r
 proc_seas_adj("Rezession", "CH")
 ```
 
-##### Reading and Writing
+### Step 4: combining ###
+
+Once the data for each keyword has been processed we end up with multiple time series - one for each keyword within the indicator.
```
Comment (a1exgo) [inline vignettes/daily-series.Rmd:123]:
```suggestion
Once the raw data for each keyword has been processed we end up with multiple time series - one for each keyword within the indicator.
```

File: vignettes/daily-series.Rmd [hunk 3]
```diff
@@ -14,143 +14,140 @@ knitr::opts_chunk$set(
...
+To turn several provided keywords into a single time series the first principal component is used.
 
-There are two special helper functions that are useful when working with production data:
+In order to achieve this the script calls `ts_prcomp()` function from the `tsbox` package.
 
-- `read_keyword`
-- `write_keyword`
+### Step 5: writing the data ###
 
+Finally the prepared index is saved to a file:
```
Comment (a1exgo) [inline vignettes/daily-series.Rmd:131]:
```suggestion
Finally the prepared index is saved to a file in the data folder:
```

File: vignettes/daily-series.Rmd [hunk 4]
```diff
@@ -14,143 +14,140 @@ knitr::opts_chunk$set(
...
+```
+write_keyword(prepared_data, "indicator", "CH")
+```
 
 
+## Summary ##
 
+In summary:
 
+- To add new indicator "indicator": create a new script at `inst/script/indicator.R`.
```
Comment (a1exgo) [inline vignettes/daily-series.Rmd]:
```suggestion
- To add a new indicator "indicator": create a new script at `inst/script/indicator.R`.
```

File: vignettes/daily-series.Rmd [hunk 5]
```diff
@@ -14,143 +14,140 @@ knitr::opts_chunk$set(
...
+Once the combination of useful keywords is selected we might want to use them as economic indicators in production and display them on [trendecon.org](https://www.trendecon.org/). This vignette describes the steps for adding new indicators to the production processes of "Trendecon".
 
-## Directory structure
+## Directory structure ##
 
-The `proc_` functions write to the working directory. If you are unsure about your current working directory, use `getwd()`.
+The names of production functions within the `trendecon` package start with `proc_`.
+They usually operate on the file system which means that calling these functions will write files to disk.
+The `proc_` functions will save files within two folders: `data` and `raw`, which are located under the working directory (`getwd()`).
+`raw` folder holds all the data downloaded from Google, plus some transformations, such as seasonally adjusted or combined series.
```
Comment (a1exgo) [inline vignettes/daily-series.Rmd:24]:
```suggestion
`raw` folder holds all the data downloaded from Google that are automatically transformed by the `trendecon`package. The transformations include seasonal adjustments or combining multiple keywords related to the indicator.
```

File: vignettes/daily-series.Rmd [hunk 6]
```diff
@@ -14,143 +14,140 @@ knitr::opts_chunk$set(
...
 
-## Initial Download
+## Initial download ##
 
-To include a new series, each keyword must be initiated. Be careful, as this causes a lot of queries to Google.
+To include a new series, each keyword must first be initiated.
+Our goal is to produce long daily time series, ideally from 2006.
+However, Google does not provide daily or weekly data for such a long time period.
+Hence we need to circumvent the problem by applying a moving window of daily and weekly queries over the whole time period.
+So be careful, as this causes a lot of queries to Google.
```
Comment (a1exgo) [inline vignettes/daily-series.Rmd:49]: This line made me insecure. You could add, why we should careful (blocking by the API?And yes, for how long?) Or could we delete it?

**Review by a1exgo — CHANGES_REQUESTED** [review]

File: vignettes/daily-series.Rmd [hunk 7]
```diff
@@ -101,6 +91,7 @@ proc_keyword_latest("Rezession", "CH")
 ```
 
 This function downloads raw daily, weekly, and monthly data for the specified pair of keyword and geo location.
+If the data for a particular keyword is not yet available, `proc_keyword_init()` should be called instead.
```
Comment (a1exgo) [inline vignettes/daily-series.Rmd:94]: the function downloaded time series (d,w,m) for the keyword "Insvolvenz". The function above, proc_keyword_latest I could not execute "could not find function "proc_keyword_latest". I guess this is to come?

**Review by a1exgo — APPROVED** [review]
Hi @KKPMW , I looked into both commits and did some comments in #48 , as I was not able to review this one. Please let me know if I can help you further out.

File: vignettes/daily-series.Rmd (hunk 5 above)
Comment (a1exgo) [reply vignettes/daily-series.Rmd:24]: I see! So there are the raw time series, and on top some with transformations if called. My bad. I think the sentence you had written is good.

File: vignettes/daily-series.Rmd (hunk 6 above)
Comment (a1exgo) [reply vignettes/daily-series.Rmd]: I like the small addition!

File: vignettes/daily-series.Rmd (hunk 7 above)
Comment (a1exgo) [reply vignettes/daily-series.Rmd:94]: great! thanks for the explanation and hint!

**Comment by KKPMW** [comment]
Hello @a1exgo , if the pull request seems more or less OK to you now, then I will merge it. Of course after that we can still work on it if we see possible improvements.

**Pull request merged**